### PR TITLE
Add text about RFC6973 to intro

### LINKED ISSUE #13: Add references to RFC 6973
We should explain the relationship to RFC 6973, the previous IAB document on privacy.

### PR DESCRIPTION
**Pull request by mirjak**
fixes #13

### CODE REVIEW

File: draft-iab-privacy-partitioning.md [hunk 1]
```diff
@@ -81,6 +81,14 @@ is no link between user-specific identity and user-specific data. However, appli
 privacy partitioning helps ensure that user privacy violations becomes more technically difficult
 to achieve over time.
 
+RFC6973 already discusses the use of identity management systems for data minimization.
```
Comment (tfpauly) [inline draft-iab-privacy-partitioning.md]:
```suggestion
{{?RFC6973}} already discusses data minimization, especially in the context of
user identity and identity management systems.
```

File: draft-iab-privacy-partitioning.md [hunk 2]
```diff
@@ -81,6 +81,14 @@ is no link between user-specific identity and user-specific data. However, appli
...
+In these systems usually an identify provider issues credentials that can be used to access a
+service without revealing the user's identity by relying on the authentication assertion from
+the identity provider (see {{Section 6.1.4 of RFC6973}}). This describes a specific form of
+privacy partitioning, simlarly as used for privacy pass (see Section {{privacypass}}).
+Privacy partitioning as defined in this document is broader by considering different deployment
```
Comment (tfpauly) [inline draft-iab-privacy-partitioning.md:89]:
```suggestion
Privacy partitioning as defined in this document goes further, to consider different deployment
```

File: draft-iab-privacy-partitioning.md (hunk 1 above)
Comment (mirjak) [inline draft-iab-privacy-partitioning.md:84]:
```suggestion
RFC6973 discusses the use of identity management systems for data minimization.
```

File: draft-iab-privacy-partitioning.md [hunk 3]
```diff
@@ -81,6 +81,14 @@ is no link between user-specific identity and user-specific data. However, appli
 privacy partitioning helps ensure that user privacy violations becomes more technically difficult
 to achieve over time.
 
+RFC6973 already discusses the use of identity management systems for data minimization.
+In these systems usually an identify provider issues credentials that can be used to access a
+service without revealing the user's identity by relying on the authentication assertion from
+the identity provider (see {{Section 6.1.4 of RFC6973}}). This describes a specific form of
+privacy partitioning, simlarly as used for privacy pass (see Section {{privacypass}}).
```
Comment (tfpauly) [inline draft-iab-privacy-partitioning.md:88]: I'm not sure this identity provider detail belongs in our intro? I think it is different than the privacy pass approach — the RFC6973 text seems to be more about only collecting what you need for an account, etc.

**Review by tfpauly — CHANGES_REQUESTED** [review]

File: draft-iab-privacy-partitioning.md [hunk 4]
```diff
@@ -86,6 +86,15 @@ of privacy partitioning, including OHAI, MASQUE, Privacy Pass, and PPM. This doc
 work in those groups and describes a framework for reasoning about the resulting privacy posture of different
 endpoints in practice.
 
+{{?RFC6973}} discusses data minimization, especially in the context of
+user identity and identity management systems.
+In these systems usually an identify provider issues credentials that can be used to access a
+service without revealing the user's identity by relying on the authentication assertion from
+the identity provider (see {{Section 6.1.4 of RFC6973}}). This describes a specific form of
+privacy partitioning, simlarly as used for privacy pass (see Section {{privacypass}}).
```
Comment (mirjak) [inline draft-iab-privacy-partitioning.md:94]:
```suggestion
privacy partitioning, similar as used for privacy pass (see Section {{privacypass}}).
```

File: draft-iab-privacy-partitioning.md [hunk 5]
```diff
@@ -86,6 +86,15 @@ of privacy partitioning, including OHAI, MASQUE, Privacy Pass, and PPM. This doc
...
 endpoints in practice.
 
+{{?RFC6973}} discusses data minimization, especially in the context of
+user identity and identity management systems.
+In these systems usually an identify provider issues credentials that can be used to access a
+service without revealing the user's identity by relying on the authentication assertion from
+the identity provider (see {{Section 6.1.4 of RFC6973}}). This describes a specific form of
+privacy partitioning, similar as used for privacy pass (see Section {{privacypass}}).
+Privacy partitioning as defined in this document goes further, to consider different deployment
+models that can create multiple contexts where data is miminized in each context.
```
Comment (mirjak) [inline draft-iab-privacy-partitioning.md:96]:
```suggestion
models that can create multiple contexts where data is minimized in each context.
```

**Review by tfpauly — APPROVED** [review]

**Pull request merged**